examples/swash_render/src/main.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright 2024 the Parley Authors
```

Comment:
Is this duplicated intentionally?

Reply:
No. This file is the correct one. The other file is a rebase conflict.